recipes/cctools-ld64/meta.yaml

```
@@ -0,0 +1,138 @@
+      - 280-cctools-missing_includes.patch
+      - 290-cctools-error_as_weak_symbol.patch
+      - 300-cctools-undef___unused_for_sysctl.patch
+      - 340-cctools-win_O_BINARY.patch
+      - 350-cctools-win_fileio_mode.patch
+      - 360-cctools-win_TMPDIR_to_TEMP.patch
+      - 370-cctools-win_execute.patch
+      - 380-cctools-win_avoid_mmap_ofile.patch
+      - 400-cctools-_WIN64-fixes.patch
+      - 410-cctools-remove_inc_arch_sparc_reg_h_PC.patch
+      - 420-cctools-autoconfiscate.patch
+  - url: https://opensource.apple.com/tarballs/ld64/ld64-{{ ld64_version }}.tar.gz
+    sha256: {{ ld64_sha256 }}
+    folder: cctools/ld64
+    patches:
+      - 100-ld64-add_sdkroot_headers.patch
+      - 150-ld64-allow_glibc_or_bsd_qsort_r.patch
+      - 180-ld64-add_CROSS_SYSROOT.patch
+      - 200-ld64-add_typename.patch
+      - 280-ld64-missing_includes.patch
+      - 320-ld64-extern_C_log2_only_if___APPLE__.patch
+      - 330-ld64-extern_C___assert_rtn.patch
+      - 340-ld64-win_O_BINARY.patch
+      - 350-ld64-win_fileio_mode.patch
+      - 390-ld64-_WIN64-fixes.patch
+      - 420-ld64-autoconfiscate.patch
+      - 500-ld64-add-fake-CrashReporterClient.h.patch
+      - 520-ld64-fix-usr-local-and-usr-ordering.patch
+      - 530-ld64-add-conda-specific-env-vars-to-modify-lib-search-paths.patch
+      - 540-ld64-add-LD64_LOG_HASH_TABLE.patch
+      - 550-ld64-fix-errorneous-tapi-version-condition.diff
+  # We copy headers necessary for building cctools/ld64 from dyld and MacOSX10.9.sdk (on all OSes).
+  # older versions of these files are added by 100-add_sdkroot_headers.patch, we overwrite those...
+  #- url: https://opensource.apple.com/tarballs/dyld/dyld-{{ dyld_version }}.tar.gz
+  #  sha256: {{ dyld_sha256 }}
+  #  folder: bootstrap/dyld
+
+build:
+  number: 0
+  skip: True  # [not osx]
+  ignore_run_exports:
+    - zlib
+  missing_dso_whitelist:
+    - /usr/lib/libxar.1.dylib  # [osx]
+
+requirements:
+  build:
+    # we use selectors rather than jinja2 here because we only want to use the native
+    #   compiler to build llvm and clang (which are natively cross-compilers.)
+    # We might want to split the runtime libs below into a separate recipe,
+    #    so that this recipe can use jinja2 instead.
+    - gcc_{{ native_compiler_subdir }}  # [linux]
+    - gxx_{{ native_compiler_subdir }}  # [linux]
+    - clang_bootstrap_osx-64  # [osx]
+    - autoconf
+    - automake
+    - make
+  host:
+    #- xar
+    # We only use the static library from this and only get away with that as it depends on nothing.
+    - zlib
+    - llvmdev
+    - libuuid   # [linux]
+    - tapi
+
+outputs:
+  - name: cctools
+    version: {{ cctools_version }}
+    script: install-cctools.sh
+    test:
+      commands:
+        - test -f $PREFIX/libexec/as/x86_64/as
+        - test -f $PREFIX/bin/{{ macos_machine }}-as
+    about:
+      home: https://opensource.apple.com/source
+      license: Apple Public Source License 2.0
```

SPDX identifier For consistency with other recipes
```suggestion
      license: APSL-2.0
```